src/process/status.cr

```diff
@@ -1,3 +1,86 @@
+{% if flag?(:win32) %}
+  require "c/ntstatus"
+{% end %}
+
+# The reason a process terminates.
```

Comment:
I think this reads a bit clearer using the past tense because at the point we have this information about a process, it has terminated. No strong feelings, though.

```suggestion
# The reason a process terminated.
```